D1371R0.pandoc

```diff
@@ -1135,70 +1146,40 @@ inspect (v) {
 }
 ```
 
-In the example above `^` is used to make former pattern to look for name `x`
+In the example above, `^` is used to make former pattern to look for name `x`
 using unqualified name lookup rules. While latter introduces new identifier `x`
 in pattern scope.
 
-Adding "introducer" for expression pattern have other advantages. They become
-apparent when compared to alternative designs below.
-
-### Using "introducer" in identifier pattern.
-
-Some languages use keywords as part of identifier pattern syntax. In
-[@SwiftPatterns] keyword `let` is used to denote identifier pattern, and the
-same keyword is used to declare bindings in general.
-
-Syntax for identifier pattern can use `let` "introducer" instead, making
-expression pattern look more like normal expression seen anywhere else in
-codebase.
-
-```cpp
-inspect(v) {
-  (1 + get_int()): std::cout << "expression matched";
-  let x: std::cout << "introduce new identifier x bound to v: " << x;
-}
-```
-
-While it makes expressions look more natural in patterns, it significantly
-limits scope of possible syntax that can be used in patterns. Expression pattern
-would contain any expression syntax, such as parentheses, arithmetic operation
-signs and etc. Any future pattern syntax would not be able to use these
-syntax elements or will cause possible ambiguity for existing code.
-
-### Not using "introducer" in any pattern.
+The problem described spans more than just *identifier pattern*. That could have
+been solved by introducer for *identifier pattern*. For example [@SwiftPatterns]
+use keyword `let` to denote identifier pattern and treat everything else as expression.
+This solution works but closes off any pattern syntax that could use syntax that is
+already possible in expression. For example *parenthesized pattern* is using
```

Comment:
What do you think about using the referring to the `1 | 2` example here again and mentioning that Swift won't be able to add `1 | 2` as a pattern later? Swift doesn't really have "parenthesized patterns" since their tuple pattern uses parentheses.